tests/Integration/Behaviour/Features/Scenario/Product/duplicate_product.feature

```diff
@@ -198,5 +244,26 @@ Feature: Duplicate product from Back Office (BO).
     And product copy_of_product1 should have identical customization fields to product1
     And product copy_of_product1 should have 1 customizable text field
     And product copy_of_product1 should have 0 customizable file fields
-#@todo: assert stock info
+    And product "copy_of_product1" should have following stock information:
+      | minimal_quantity    | 12         |
+      | quantity            | 0          |
+      | location            |            |
+      | low_stock_threshold | 42         |
+      | low_stock_alert     | true       |
+      | available_date      | 1969-07-16 |
+    And product "copy_of_product1" localized "available_now_labels" should be:
+      | locale | value      |
+      | en-US  | get it now |
+      | fr-FR  |            |
+    And product "copy_of_product1" localized "available_later_labels" should be:
+      | locale | value        |
+      | en-US  | too late bro |
+      | fr-FR  |              |
+#    no a way to check if the images are actually the ones we expect,
+#    because references are attached to id, so at least we can count them and check if legends match
+    And product "copy_of_product1" should have 2 images in total
+    And product "copy_of_product1" images should have following legends:
```

Comment:
Why can't you use the same test as for the initial product?
I guess because you can't know the ID of the duplicated images?

Reply:
yes, I don't know the ids :(

Reply:
Oh yeah because they are more complete now I guess the images are duplicated as well which is why we don't know the IDs?

Then instead of making all optional in the current full assertion maybe we can create a dedicated smaller check for this case indeed It's not a full cover but it's ok for now We might need to improve this someday if we have some bugs on the feature though 😅